contracts/Honey.sol

```diff
@@ -177,4 +183,17 @@ contract Honey is IERC20 {
 
         _transfer(from, to, value);
     }
+
+    function registerTokenOnL2(
+        address l2CustomTokenAddress,
+        uint256 maxSubmissionCost,
+        uint256 maxGas,
+        uint256 gasPriceBid
+    ) external {
+        // TODO: Do we need this?
```

Comment:
Yes, allow us to permissionless register our token to the Gateway without having to wait for an admin call. From the docs, point 2: https://developer.offchainlabs.com/docs/bridging_assets#setting-up-your-token-with-the-generic-custom-gateway

Reply:
We can deploy our own CustomGateway and `forceRegister()` it though so it's not strictly necessary.